File: go-solutions/26.go
package main

func removeDuplicates(nums []int) int {
	index := 1
	for i := 1; i < len(nums); i++ {
		if nums[i] != nums[i-1] {
			nums[index] = nums[i]
			index++
		}
	}
	return index
}

File: cpp/26.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int pos = 0;
        for (int i=0; i<nums.size(); i++){
            if(i==0 || nums[i] != nums[pos-1]){
                nums[pos++] = nums[i];
            }
        }
        return pos;
    }
};